packages/styleguide/src/components/Colors/ColorContext.tsx

```diff
@@ -24,16 +19,18 @@ const createScheme = (specificColors) => {
     ...specificColors,
   }
 
-  const accessCSSColor = colorDefinitions.cssColors
-    ? (color) =>
-        colorDefinitions.cssColors[color] || colorDefinitions[color] || color
-    : (color) => colorDefinitions[color] || color
-
   const { mappings = {} } = colorDefinitions
 
   const getCSSColor = (color, mappingName = undefined) => {
+    // For backwards compatibility, we map some colors to variable names.
+    // E.g. when mappingName is 'format', '#000' is looked up in mappings.format['#000'] and becomes 'accentColorMeta'
     const mapping = mappings[mappingName] || {}
-    return accessCSSColor(mapping[color] || color)
+
+    return color in mapping
+      ? `var(--color-${mapping[color]})`
+      : color in colorDefinitions
+      ? `var(--color-${color})`
```

Comment:
🥴

Reply:
Improved in the [sequel](https://github.com/republik/plattform/pull/528/commits/ffde21a4176e85cfe683015aed2a33a24a6801f7#diff-a7d5b1b2b5d16e2da4dc28c711272ec8a626946f1579e93354156c89f285572fR15-R17)